Accredited bodies round 3 placeholder added (#70)
DFE-Digital/bat-design-history: app/posts/publish-teacher-training-courses/2019-11-01-accredited-bodies-research-round-3.md

---
title: Accredited bodies research - Round 3
description: Research on the role of accredited bodies and their relationships with their partner schools
date: 2019-11-01
related:
  items:
  - text: Lookback sessions
    href: https://lookback.io/dfe-digital/find-accrediting-bodies-round-3
  - text: Research playback
    href: https://docs.google.com/presentation/d/1tt_QwL2cU7fpZ9pg4JywENfJGp66DBWtkfTqiBYsLEY/edit#slide=id.g75932c5169_0_62
---
Round 3 is a continuation of the research started in [Accredited Bodies Research Round 1](/publish-teacher-training-courses/accredited-bodies-research-round-1) and [Round 2](/publish-teacher-training-courses/accredited-bodies-research-round-2). This round examines the role of accredited bodies and their relationships with their partner schools. 

## Findings

> * At the start of each recruitment cycle, UCAS compiles list of reference data for any providers to consume. This is done via XML or ODBC into the provider’s SRS system. Any changes after this are not automatically synced.

Reviewer: ```suggestion
* At the start of each recruitment cycle, UCAS compiles a list of reference data for providers to consume. This is done via XML or ODBC into the provider’s SRS system. Any changes after this are not automatically synced.
```

> * Accrediting body organisations’ needs vary widely. HEIs and SCITTs that are part of HEIs have SRS. Other SCITTs will use UCAS Weblink and manual processes in order to process applications. Those with SRS are affected by systems not being in sync.

Reviewer: Can you reword this point, it does read clearly.

* The staging area is an area within the SRS where applications sit to be processed. Applications are stuck in staging because the application cannot be matched to a course in the system. Providers get an import error list, where details about the candidate are present but nothing about the course.To fix this, they typically will log onto UCAS Weblink to find the application, note the details and then go onto Find to find the course to set up in their SRS.
* Providers have very different student record systems - there is the base application but much of the features are personalised to the organisation.

> * Organisations map out codes themselves e.g. Location in School A on UCAS will be assigned to Code X in one organisation’s SRS and Code Z in a different organisation with the same location. This can make integration difficult.

Reviewer: ```suggestion
* Organisations map out codes themselves eg Location in School A on UCAS will be assigned to Code X in one organisation’s SRS and Code Z in a different organisation with the same location. This can make integration difficult.
```
Style nit

* Providers need CSVs for multiple reasons. They need to check against allocations spreadsheets, use spreadsheets to share data themsleves as there is no single access point and alos for data manipulation.

### Why do accredited bodies need course information  

> _"We need to know because we are accountable for all the courses that we accredit. There is a reputational risk if we don’t have access [to the information]."_

Reviewer: `>` will make a blockquote for quotes

```suggestion
> "We need to know because we are accountable for all the courses that we accredit. There is a reputational risk if we don’t have access [to the information]." 
```

Participants have said it would be useful to have cross checking abilities between allocations and the data about the courses they accredit.

_"Do you (DfE) cross check that information [courses live] against the allocations or is it just what people have put on Publish?From an admissions worry point, if we see a course that we don’t expect to see there, we wonder if they (School Direct) have been in touch with DfE, with our side, have they gone through the right channels?"_

### UCAS Weblink

The UCAS Weblink is used by both accredited bodies with SRS and accredited bodies without SRS to manage applications from candidates. It lists all the providers and the applications that have come through for those partners and what status the applications are though tt has it's limitations. ABs will only see the details about what is being applied to, and not all the courses they are accredited bodies for.

_"For us to go in and check the data for our partners, we’d have to go in the same way as an applicant would and that really isn’t an efficient way of working."_

> 

Reviewer: (Lots of trailing whitespace)
